test/verify/check-storage-multipath

```diff
@@ -51,6 +51,14 @@ class TestStorage(StorageCase):
                     return False
             return True
 
+        # At least on Fedora 27, multipath only looks at SCSI_IDENT_
+        # and ID_WWN properties, so we install a custom udev rule to
+        # set ID_WWN to something that can identify multipathed devices.
+        #
+        m.write("/etc/udev/rules.d/99-fake-wwn.rules", 'SUBSYSTEM=="block", ENV{ID_WWN}="$env{ID_SCSI_SERIAL}"\n')
```

Comment:
@mvollmer: Shouldn't this rule be limited to the created fake device (I assume there is one)? Like that it will modify *all* block devices, which isn't a realistic test environment.

Reply:
Also, is this only an artifact of faking some devices in the tests, or does it hide some actual bug?

Reply:
> Shouldn't this rule be limited to the created fake device (I assume there is one)?

We could restrict the rule to devices that actually have ID_SCSI_SERIAL and no ID_WWN, to avoid setting empty strings or clobbering existing values.  But I don't think it makes a practical difference for this one test.

> is this only an artifact of faking some devices in the tests

Yes, I would say so.  Multipathed emulated block devices are pretty pointless, so I guess people don't use them outside of tests.

Reply:
Some IRC conversation clarified things. The two drives get attached to QEMU from outside, and I previously missed the `udevadm trigger` below (FTR, this is an unnecessarily big hammer, `--subsystem-match=block` would have been enough. But as long as that doesn't cause troubles, fine for me).